internal/printer/print-to-json.go

```diff
@@ -230,7 +230,7 @@ func renderNode(p *printer, parent *ASTNode, n *Node, opts t.ParseOptions) {
 				}
 				position := attrPositionAt(p, &attr, opts)
 				raw := ""
-				if attr.Type == QuotedAttribute {
+				if attr.Type == QuotedAttribute || attr.Type == TemplateLiteralAttribute {
```

Comment:
Updated this to also include *some* of the raw unterminated template literal attribute value